tests-perf/individual_emails/individual-emails.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+from dotenv import load_dotenv
+from locust import HttpUser, constant_pacing, task
+
+load_dotenv()
+
+
+class NotifyApiUser(HttpUser):
+    wait_time = constant_pacing(0.3)  # about 200 / minute hopefully
```

Comment:
Not sure if the configuration is right here. Can you tell me more about the desired goal? Why not use constant_throughput of 1 minute for example with a number of users set to 200 for example?

Reply:
Using 200 users hitting every minute means we get a big spike at the 1 minute marks and nothing in between. I'm trying to more closely emulate what happens in prod, where a service will send constantly at a high rate.

There is a locust pacing function `constant_throughput`, but it isn't available in the version of locust we use here, and we can't bump locust up because we're using an old flask 😞 

It might be useful to move all our performance / smoke testing code to its own repo where we are freer to install packages just needed by the tests.

Reply:
That might be another case for upgrading werkzeug? so we can upgrade flask.

Reply:
Why not set the constant_pacing to every second though? so we get an easy to calculate and to expect formula and a relatively constant throughput?